cloudformation/database/serverless-database.yaml

```diff
@@ -0,0 +1,345 @@
+  ServerlessDatabaseClusterSubnetGroup:
+    Type: 'AWS::RDS::DBSubnetGroup'
+    Properties:
+      DBSubnetGroupDescription: !Sub Subnets for ServerlessDatabaseCluster-${AWS::Region} database
+      SubnetIds:
+        - !If [CreateSubnets, !Ref ServerlessDatabaseClusterPrivateSubnet1, !Ref PrivateSubnetOne]
+        - !If [CreateSubnets, !Ref ServerlessDatabaseClusterPrivateSubnet2, !Ref PrivateSubnetTwo]
+      Tags:
+        - Key: 'parallel-cluster:accounting'
+          Value: subnet-group
+        - Key: 'parallel-cluster:accounting:scheduler'
+          Value: slurm
+        - Key: 'parallel-cluster:accounting:version'
+          Value: '1.0'
+  ServerlessDatabaseClusterSecurityGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      GroupDescription: RDS security group
+      SecurityGroupEgress:
+        - CidrIp: 0.0.0.0/0
+          Description: Allow all outbound traffic by default
+          IpProtocol: '-1'
+      Tags:
+        - Key: 'parallel-cluster:accounting'
+          Value: database-security-group
+        - Key: 'parallel-cluster:accounting:scheduler'
+          Value: slurm
+        - Key: 'parallel-cluster:accounting:version'
+          Value: '1.0'
+      VpcId: !If [CreateVPC, !Ref ServerlessDatabaseClusterVpc, !Ref Vpc]
+  ServerlessDatabaseClusterSecurityGroupInboundRule:
+    Type: 'AWS::EC2::SecurityGroupIngress'
+    Properties:
+      IpProtocol: tcp
+      Description: Allow incoming connections from client security group
+      FromPort: !GetAtt
+        - ServerlessDatabaseCluster
+        - Endpoint.Port
+      GroupId: !GetAtt
+        - ServerlessDatabaseClusterSecurityGroup
+        - GroupId
+      SourceSecurityGroupId: !GetAtt
+        - ServerlessClusterDatabaseClientSecurityGroup
+        - GroupId
+      ToPort: !GetAtt
+        - ServerlessDatabaseCluster
+        - Endpoint.Port
+  ServerlessDatabaseCluster:
+    Type: 'AWS::RDS::DBCluster'
+    Properties:
+      Engine: aurora-mysql
+      CopyTagsToSnapshot: true
+      DBClusterParameterGroupName: !Ref ServerlessDatabaseClusterParameterGroup
+      DBSubnetGroupName: !Ref ServerlessDatabaseClusterSubnetGroup
+      EnableHttpEndpoint: false
+      EngineMode: serverless
+      MasterUsername: !Ref ClusterAdmin
+      MasterUserPassword: !Sub '{{resolve:secretsmanager:${AdminPasswordSecret}:SecretString:::}}'
+      ScalingConfiguration:
+        AutoPause: true
+        MaxCapacity: !Ref MaxCapacity
+        MinCapacity: !Ref MinCapacity
+        SecondsUntilAutoPause: 600
+      StorageEncrypted: true
+      Tags:
+        - Key: 'parallel-cluster:accounting'
+          Value: database
+        - Key: 'parallel-cluster:accounting:scheduler'
+          Value: slurm
+        - Key: 'parallel-cluster:accounting:version'
+          Value: '1.0'
+      VpcSecurityGroupIds:
+        - !GetAtt
+          - ServerlessDatabaseClusterSecurityGroup
+          - GroupId
+    UpdateReplacePolicy: Snapshot
+    DeletionPolicy: Snapshot
```

Comment:
I understand the reasoning behind `DeletionPolicy: Snapshot` and `UpdateReplacePolicy: Snapshot`. Nevertheless, the template is meant to be used for tutorials and beta testing, so we do not want to introduce extra cost for users once the stack will be deleted.
I think all the resources should be deleted at stack deletion.

Reply:
Changed to `Delete`.